support-frontend/app/controllers/WeeklySubscriptionController.scala

```diff
@@ -42,11 +43,16 @@ class WeeklySubscriptionController(
   def weekly(countryCode: String, orderIsAGift: Boolean): Action[AnyContent] = CachedAction() { implicit request =>
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
     val canonicalLink = Some(buildCanonicalWeeklySubscriptionLink("uk", orderIsAGift))
+    // We're currently running different default promos in different regions
+    val defaultPromotion =
+      if (countryCode == "AU" || countryCode == "NZ") ausPromotion
+      else francePromotion
+
     val queryPromos =
       request.queryString
         .get("promoCode")
         .map(_.toList)
-        .getOrElse(List(DefaultPromotions.GuardianWeekly.NonGift.ausPromotion))
+        .getOrElse(List(defaultPromotion))
```

Comment:
although the `landingCopyProvider` takes a `List` of promos, it calls `.headOption` and only uses the first one! So we have to pick the right one here based on countryCode